buildscripts/kokoro/gae-interop.sh

```diff
@@ -1,3 +1,75 @@
-#!/bin/sh
+#!/bin/bash
 
-# An empty placeholder shellscript
+set -exu -o pipefail
+
+pwd
+hostname
```

Comment:
It looks like these are leftover commands from exploring Kokoro. These should be removed.

Reply:
Fixed